### PR TITLE
Work on forward/backward ref's support

### PR DESCRIPTION
**Pull request by deniskoronchik**
# Issue

#4618 

## Tasklist
 - [x] review
 - [x] adjust for comments

### CODE REVIEW

File: features/car/route_relations.feature [hunk 1]
```diff
@@ -74,28 +74,28 @@ Feature: Car - route relations
...
+            | route       | west      | ab          | road  | 80  | US:I    |
+
+        And the relations
+            | type        | direction | way:backward | route | ref | network |
+            | route       | east      | ab           | road  | 80  | US:I    |
+
+        When I route I should get
+            | waypoints | route              | ref                    |
+            | a,b       | mainroad,mainroad  | I 80 $west,I 80 $west  |
+            | b,a       | mainroad,mainroad  | I 80 $east,I 80 $east  |
```
Comment (deniskoronchik) [inline features/car/route_relations.feature]: @danpat Check if this test correct. Seems there was an error, that i've fixed

Comment (TheMarex) [reply features/car/route_relations.feature]: I agree your version makes more sense.

**Comment by TheMarex** [comment]
@deniskoronchik Can you add a changelog entry for this? Thanks.

**Comment by deniskoronchik** [comment]
Will do that

**Pull request merged**